bin/git-sync-mirror.sh

```diff
@@ -180,15 +180,22 @@ if ! whoami &> /dev/null; then
 fi
 
 # Create local repositories
-LOCAL_REPO_SRC="$(mktemp -d)"
-LOCAL_REPO_DST="$(mktemp -d)"
+LOCAL_REPO_SRC="${LOCAL_REPO_SRC:-"$(mktemp -d)"}"
+LOCAL_REPO_DST="${LOCAL_REPO_DST:-"$(mktemp -d)"}"
+
+if [ -z "$(ls -A ${LOCAL_REPO_SRC})" ]; then
+     clone_local_repo "${SRC_REPO}" "${LOCAL_REPO_SRC}"
+fi
+
+if [ -z "$(ls -A ${LOCAL_REPO_DST})" ]; then
+     clone_local_repo "${DST_REPO}" "${LOCAL_REPO_DST}"
+fi
```

Comment:
these checks and `clone_local_repo` calls have to move below the git config section again. Otherwise we'll loose a lot of functionality related with ssl verification and http proxy setup.. Is there a reason why you moved this up here?

Reply:
No, there is no specific reason for the placement. The only reason is that im a newbie.